orbit/cmd/orbit/orbit.go

```diff
@@ -350,6 +356,12 @@ func main() {
 			return fmt.Errorf("cleanup old files: %w", err)
 		}
 
+		log.Info().Msg("running single query for UUID")
+
+		uuidStr, _ := getUUID(osquerydPath)
+		log.Info().Msg("UUID is")
```

Comment:
No need for two separate statements, this can just be `log.Info().Msgf("UUID is=%s", uuidStr)`